core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/impl/api/toDataFrame.kt

```diff
@@ -38,20 +39,37 @@ import kotlin.reflect.jvm.javaField
 import kotlin.reflect.typeOf
 
 private val valueTypes = setOf(
+    Any::class,
```

Comment:
Uhm, `Any` doesn't cause issues with other classes? I guess this doesn't mean "a subtype of", rather the exact type? Maybe mention this in kdoc for future us

Reply:
User can provide `Iterable<Any?>`.

Reply:
And yeah, these are exact types.

Reply:
so now if a user has something like `listOf(cat, dog, house, table).toDataFrame()` they will get a `value` column, right?